Upstream scintilla patch.
Avoid double-paint issues on the Mac.
Bug 97330, bug 97214.
Index: /Users/toddw/as/komodo-devel/src/scintilla/cocoa/ScintillaView.mm
===================================================================
--- cocoa/ScintillaView.mm	(revision 78637)
+++ cocoa/ScintillaView.mm	(working copy)
@@ -129,7 +130,7 @@
   CGContextRef context = (CGContextRef) [[NSGraphicsContext currentContext] graphicsPort];
   
   if (!mOwner.backend->Draw(rect, context)) {
-    [self display];
+    [self setNeedsDisplay:YES];
   }
 }
 
